### DISCUSSION TITLE
Display SHT1106 not working with Tasmota-Displays 14.0.0

### DISCUSSION THREAD

**Discussion by topsurfer** [arendst/Tasmota#21489] [Q&A] [answered]
Hi,
I find out that my display SHT1106 not working with Version 14.0.0 as the, work in older versions.
Using same settings and commands, but Display is always dark.

Install Version 9.2 for testing and it was working immedialtly.

Is this a Bug or need the display(s) other configuration?

**Comment by sfromis** [comment, marked as answer]
With version [14.0.0](https://github.com/arendst/Tasmota/releases/tag/v14.0.0), a significant change is that the old display-specific display drivers are replaced by a new "Universal Display Driver", which has to be configured by display type. Migration instructions on this page:
https://tasmota.github.io/docs/Universal-Display-Driver/

**Comment by topsurfer** [comment]
Thx for this info.

I have the need to use Display and Sensors, before I use Gitpod.io and add (needed) Display into the Sensors-Firmware.
Will this also work with Universal Driver, is there some page with hints or a "how to" ? Thx!

**Reply by sfromis** [reply]
Adding to the sensors firmware is already not a standard way of doing builds. Still, the page I linked does say what `#define` to add when you compile yourself.

**Comment by DrGalvany** [comment]
Hi, sorry to jump in here, but I have the same display and problem.
Unfortunately, I'm not that familiar with the subject and my English is not that good either.
Unfortunately, I don't understand what I have to do here.
I have looked at the page and I don't understand anything.
Is there an exact description or video somewhere of what I have to do here?
Or, can someone please explain it to a layman like me?
I would be very pleased.
In German would be a dream. :)

Thanks, Michael


Hallo, entschuldigt, dass ich mich hier einklinke, aber ich habe das gleiche Display und Problem.
Ich bin mit dem Thema leider nicht so konform und mein Englisch ist leider auch nicht so gut.
Ich verstehe es bedauerlicherweise nicht, was ich hier machen muss.
Die Seite habe ich mir angesehen und verstehe nur Bahnhof.
Gibt es irgendwo ein genau Beschreibung oder Video, was mach hier machen muss?
Bzw. kann das mir das jemand Bitte für einen Laien, wie mir erklären.
Würde mich sehr freuen.

Auf Deutsch wäre ein Traum :)

Danke Michael

**Reply by sfromis** [reply]
Did you try using "Google Translate" on the docs page linked in my response? While machine translations are not "perfect", it does look like a reasonable German translation from the tool.

**Reply by DrGalvany** [reply]
Hello,
of course, I did :)
But I don't understand step 5.
I have saved the corresponding “display.ini”, but from now on, I don't know how to proceed, this is entirely new territory and I haven't had any contact with this topic yet.
That's why I would appreciate it if you could show me (support) a more precise procedure of what exactly needs to be done.

Many thanks in advance.

Michael
